dev/rpm/build-rpm-updates.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
@charlespierce do we have someone familiar with RPM who could give this some attention? I'm looking through it, but I have literally never done *anything* with RPM (and almost nothing with RHEL), so I'd very much prefer not to be the only reviewer on this!

Reply:
Most of this (and the below `volta-updates.spec`) were copied from `build-rpm.sh` and `volta.spec`, then adapted for the necessary changes (since we can't apply Rust feature flags to shell scripts). So the easiest way to review them is probably to diff with those files.

That said, @mikrostew wrote the originals, so he probably has the best context for review.

Reply:
OHHHH. That's good to know! Thanks!